src/Mvc/.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It's strange that we only have this here and not in the other directories.

Would recommend removing it and including notes in the `BuildFromSource.md` about what extensions we recommend people use.

Reply:
Deleted `extensions.json` and added instructions to `BuildFromSource.md` for recommended extension!